templates/listing-page.twig

```diff
@@ -1,7 +1,11 @@
 <div class="page-content container" id="content">
     <div class="wp-block-separator is-style-wide listing-page-separator"></div>
     <div class="d-flex align-items-center justify-content-between listing-page-title">
-        <h2>{{ listing_page_title ?? __( 'All articles', 'planet4-master-theme' ) }}</h2>
+        {% if fn('is_home') %}
```

Comment:
Can you explain to me this condition?
In which scenario can I see the `else` statement as true?

Reply:
This is a bit confusing on WP's part, but [is_home()](https://developer.wordpress.org/reference/functions/is_home/) what we can use to check that this is in fact the Posts Page and not another listing page (tag, category, etc) We use the same check [here](https://github.com/greenpeace/planet4-master-theme/blob/main/src/ListingPage.php#L47)

Reply:
Ohh, I see, thanks!
Maybe adding a comment explaining it will be good enough.